src/cmd/create.go

```diff
@@ -0,0 +1,490 @@
+	homeCanonical, err = filepath.EvalSymlinks(homeEnv)
+	if err != nil {
+		logrus.Fatalf("Failed to canonicalize %s", homeEnv)
+	}
+	logrus.Infof("Canonicalized %s to %s", homeEnv, homeCanonical)
+
+	logrus.Info("Checking if /home is a symbolic link to /var/home")
+	homeSymPath, err := filepath.EvalSymlinks("/home")
+	if err != nil {
+		logrus.Error("Failed to evaluate if /home is a symbolic link")
+	}
+	if homeSymPath == "/var/home" {
+		logrus.Info("/home is a symbolic link to /var/home")
+		command = append(command, "--home-link")
+	}
+
+	conn, err := dbus.SessionBus()
+	if err != nil {
+		logrus.Error("Failed to connect to Session Bus")
+	}
+	defer conn.Close()
+
+	logrus.Info("Calling org.freedesktop.Flatpak.SessionHelper.RequestSession")
+	SessionHelper := conn.Object("org.freedesktop.Flatpak", "/org/freedesktop/Flatpak/SessionHelper")
+	call := SessionHelper.Call("org.freedesktop.Flatpak.SessionHelper.RequestSession", 0)
+	if call.Err != nil {
+		logrus.Fatal("Failed to call org.freedesktop.Flatpak.SessionHelper.RequestSession")
+	}
+
+	// Add the environment variables that hold a value
+	for _, env := range preservedEnvVars {
+		value := viper.GetString(env)
+		if len(value) != 0 {
+			createArgs = append(createArgs, fmt.Sprintf("--env=%s=%s", env, value))
+		}
+	}
+
+	createArgs = append(createArgs, []string{
+		"--volume", fmt.Sprintf("%s:/usr/bin/toolbox:ro", viper.GetString("TOOLBOX_PATH")),
+		"--volume", fmt.Sprintf("%s:%s", viper.GetString("XDG_RUNTIME_DIR"), viper.GetString("XDG_RUNTIME_DIR")),
+		"--volume", fmt.Sprintf("%s/.flatpak-helper/monitor:/run/host/monitor", viper.GetString("XDG_RUNTIME_DIR")),
+		"--volume", fmt.Sprintf("%s:%s:rslave", homeCanonical, homeCanonical),
+		"--volume", "/etc:/run/host/etc",
+		"--volume", "/dev:/dev:rslave",
+		"--volume", "/run:/run/host/run:rslave",
+		"--volume", "/tmp:/run/host/tmp:rslave",
+		"--volume", fmt.Sprintf("/usr:/run/host/usr:%s,rslave", usrMountDestinationFlags),
+		"--volume", "/var:/run/host/var:rslave",
+		imageName}...)
+
+	createArgs = append(createArgs, command...)
+
+	logrus.Infof("Trying to create container %s", containerName)
+	logrus.Debug(createArgs)
+
+	output, err = podman.CmdOutput(createArgs...)
+	if err != nil {
+		logrus.Debug(err)
+		logrus.Fatalf("Failed to create container %s", containerName)
+	}
+
+	fmt.Printf("Created container: %s\n", containerName)
+	fmt.Printf("Enter with: toolbox enter %s\n", containerName)
+
+	return nil
+}
+
+func findLocalToolboxImage(imageName string) bool {
+	logrus.Info("Looking for the image locally")
+
+	logrus.Infof("Looking for image %s", imageName)
+	if podman.ImageExists(imageName) {
+		return true
+	}
+
+	hasDomain := utils.ReferenceHasDomain(imageName)
+
+	if !hasDomain {
+		imageName = "localhost/" + imageName
+		logrus.Infof("Looking for image %s", imageName)
+
+		if podman.ImageExists(imageName) {
+			return true
+		}
+	}
+
+	return false
+}
+
+func checkIfToolboxImage(imageName string) (bool, error) {
```

Comment:
This looks like something that could be used to implement `rmi`. Maybe we should split it out into `pkg/utils`?